src/app/cube/details/action-panel/action-panel.component.ts

```diff
@@ -127,10 +111,20 @@ export class ActionPanelComponent implements OnInit, OnDestroy {
     this.addingToLibrary = false;
   }
 
+  async downloadRevised(download?: boolean) {
```

Comment:
This function doesn't need to be async since we aren't awaiting anything.

Reply:
Resolved with remove async.